src/board/v2/Makefile.inc

```diff
@@ -11,6 +11,11 @@ ifeq ($(USE_PROCESS_STACKSIZE),)
   USE_PROCESS_STACKSIZE = 0x738
 endif
 
+# Max tracking channels.
+ifeq ($(MAX_CHANNELS),)
+  MAX_CHANNELS = 11
```

Comment:
I would suggest to use (MAX_CHANNELS + 1) to define NAP_MAX_N_TRACK_CHANNELS value for Piksi v2 and v3.